tensorflow/lite/delegates/flex/delegate.cc

```diff
@@ -149,7 +149,7 @@ extern "C" {
 #if defined(_WIN32)
 __declspec(dllexport)
 #endif
-    tflite::TfLiteDelegateUniquePtr TF_AcquireFlexDelegate() {
-  return tflite::FlexDelegate::Create();
-}
+//    tflite::TfLiteDelegateUniquePtr TF_AcquireFlexDelegate() {
```

Comment:
what happens if you didn't comment this out?

Reply:
I get this build error:

```
ERROR: /Users/jennyabrahamson/tf/tensorflow/tensorflow/lite/delegates/flex/BUILD:77:1: C++ compilation of rule '//tensorflow/lite/delegates/flex:delegate_only_runtime' failed (Exit 1)
tensorflow/lite/delegates/flex/delegate.cc:152:37: error: 'TF_AcquireFlexDelegate' has C-linkage specified, but returns user-defined type 'tflite::TfLiteDelegateUniquePtr' (aka 'unique_ptr<TfLiteDelegate, void (*)(TfLiteDelegate *)>') which is incompatible with C [-Werror,-Wreturn-type-c-linkage]
    tflite::TfLiteDelegateUniquePtr TF_AcquireFlexDelegate() {
```

Reply:
I'll handle this separately. Could you revert this change and update your PR?

Reply:
done

Reply:
thanks. BTW, Could you share the NDK version you used?

Reply:
Yeah it's 20.0.5594570